specs/default/chef/site-cookbooks/symphony/recipes/hostfactory.rb

```diff
@@ -122,20 +122,34 @@
 #   not_if "grep -q azurecc #{hostfactory_confdir}/requestors/hostRequestors.json"
 # end
 
-template "#{hostfactory_confdir}/requestors/symA/conf/symAreq_config.json" do
-  source "#{node['symphony']['hostfactory']['templates_dir']}/symAreq_config.json.erb"
-  group "egoadmin"
-  owner "egoadmin"
-  not_if "grep -q azurecc #{hostfactory_confdir}/requestors/symA/conf/symAreq_config.json"
-end
+# template "#{hostfactory_confdir}/requestors/symA/conf/symAreq_config.json" do
+#   source "#{node['symphony']['hostfactory']['templates_dir']}/symAreq_config.json.erb"
+#   group "egoadmin"
+#   owner "egoadmin"
+#   not_if "grep -q azurecc #{hostfactory_confdir}/requestors/symA/conf/symAreq_config.json"
+# end
 
-template "#{hostfactory_confdir}/requestors/symA/conf/symAreq_policy_config.json" do
-  source "#{node['symphony']['hostfactory']['templates_dir']}/symAreq_policy_config.json.erb"
-  group "egoadmin"
-  owner "egoadmin"
-  not_if "grep -q azurecc #{hostfactory_confdir}/requestors/symA/conf/symAreq_policy_config.json"
-end
+# template "#{hostfactory_confdir}/requestors/symA/conf/symAreq_policy_config.json" do
+#   source "#{node['symphony']['hostfactory']['templates_dir']}/symAreq_policy_config.json.erb"
+#   group "egoadmin"
+#   owner "egoadmin"
+#   not_if "grep -q azurecc #{hostfactory_confdir}/requestors/symA/conf/symAreq_policy_config.json"
+# end
 
+jetpack_download "symphony/#{node['symphony']['pkg_plugin']}" do
+    project "symphony"
+end 
+bash 'Installing HostFactory...' do
+  code <<-EOH
+  cd #{node['jetpack']['downloads']}
+  unzip #{node['symphony']['pkg_plugin']}
```

Comment:
generally, i'd rather extract to /tmp - less likely to leak disk that way
I'd also recommend removing the extracted directory upon successful install